utils.go

```diff
@@ -0,0 +1,89 @@
+package cloudinary
+
+import (
+	"fmt"
+	"github.com/cloudinary/cloudinary-go/v2/api"
+	"github.com/goravel/framework/contracts/filesystem"
+	"github.com/goravel/framework/support/file"
+	"io"
+	"net/http"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+// RemoveFileExtension removes the file extension from the given filename, if it exists.
+// If there's no file extension, the original filename is kept unchanged.
+func RemoveFileExtension(filename *string) {
+	ext := filepath.Ext(*filename)
+	if ext != "" {
+		*filename = (*filename)[0 : len(*filename)-len(ext)]
+	}
+}
+
+// GetAssetType returns the asset type based on the file extension.
+func GetAssetType(file *string) api.AssetType {
+	fileName := *file
+	fileExtension := strings.ToLower(filepath.Ext(fileName))
+	// Check if the file has an extension or not.
+	if fileExtension == "" {
+		// Assuming "Image" asset type for files with no extension.
+		return api.Image
+	}
+
+	switch fileExtension {
+	case ".ai", ".avif", ".bmp", ".bw", ".djvu", ".dng", ".ps", ".ept", ".eps", ".eps3", ".fbx", ".flif", ".gif", ".glb", ".heif", ".heic", ".ico", ".indd", ".jpg", ".jpe", ".jpeg", ".jp2", ".wdp", ".jxr", ".hdp", ".jxl", ".obj", ".pdf", ".ply", ".png", ".psd", ".arw", ".cr2", ".svg", ".tga", ".tif", ".tiff", ".u3ma", ".usdz", ".webp":
+		RemoveFileExtension(file)
+		return api.Image
+	case ".3g2", ".3gp", ".avi", ".flv", ".m3u8", ".ts", ".m2ts", ".mts", ".mov", ".mkv", ".mp4", ".mpeg", ".mpd", ".mxf", ".ogv", ".webm", ".wmv", ".aac", ".aiff", ".amr", ".flac", ".m4a", ".mp3", ".ogg", ".opus", ".wav":
+		RemoveFileExtension(file)
+		return api.Video
```

Comment:
I have included all types mentioned in cloudinary docs here if there is a new type it will be considered as FIle by default (raw). [docs](https://cloudinary.com/documentation/image_transformations#image_format_support) I will look a better way if there is.

Reply:
We must be missing some types when `cloudinary` adds new types  in the future ~